src/widgets/radio.jsx

```diff
@@ -49,79 +189,54 @@ var BaseRadio = React.createClass({
+                                    this.props.multipleSelect ||
+                                    this.props.deselectEnabled;
+                                this.checkOption(
+                                    i,
+                                    shouldToggle ? !choice.checked : true);
                             }
-                            onClick={!this.props.labelWrap ? null : (e) => {
-                                // Don't send this to the scratchpad
-                                e.preventDefault();
-                                if (!this.props.apiOptions.readOnly) {
-                                    var shouldToggle =
-                                        this.props.multipleSelect ||
-                                        this.props.deselectEnabled;
-                                    this.checkOption(
-                                        i,
-                                        shouldToggle ? !choice.checked : true);
-                                }
-                            }}>
-                    <div>
-                        <span className="checkbox">
-                            <input
-                                ref={"radio" + i}
-                                type={inputType}
-                                name={radioGroupName}
-                                checked={choice.checked}
-                                onClick={(e) => {
-                                    // Avoid sending this to the parent
-                                    e.stopPropagation();
-                                }}
-                                onChange={(e) => {
-                                    this.checkOption(i, e.target.checked);
-                                }}
-                                disabled={this.props.apiOptions.readOnly} />
-                        </span>
-                        {/* A pseudo-label. <label> is slightly broken on iOS,
-                            so this works around that. Unfortunately, it is
-                            simplest to just work around that everywhere. */}
-                        <span
-                                className={
-                                    ApiClassNames.RADIO.OPTION_CONTENT + " " +
-                                    ApiClassNames.INTERACTIVE
-                                }
-                                style={{
-                                    cursor: "default",
-                                }}>
-                            {content}
-                        </span>
-                        {Exercises.cluesEnabled && this.props.showClues &&
-                            choice.checked &&
-                            <div className="perseus-radio-clue">
-                                {choice.clue}
-                            </div>}
-                    </div>
+                    }}>
+
+                    {React.createElement(( choice.isNoneOfTheAbove ?
```

Comment:
Since the Choice and ChoiceNoneAbove classes are so similar, maybe it makes sense to always use Choice here, and to pass in the necessary none of the above changes as props. For example:

    var content = choice.isNoneOfTheAbove ?
        "None of the above" :
        choice.content;

    var classSet = choice.isNoneOfAbove ? {
        "none-of-above": true
    } : {};

    <Choice
        ref={`radio${i}`}
        classSet={classSet}
        checked={choice.checked}
        clue={this.props.showClues ? choice.clue : ''}
        content={content}
        correct={choice.correct}
        disabled: this.props.apiOptions.readOnly,
        groupName: radioGroupName,
        type: inputType,
        onChecked: (checked) => {
            this.checkOption(i, checked);
        }
    />